.travis.yml

```diff
@@ -1,22 +1,26 @@
 sudo: false
 language: python
 python:
-  - "2.7"
-cache:
-  directories:
-    - $HOME/.pip-cache/
+  - '2.7'
+  - '3.4'
+  - '3.5'
```

Comment:
Could we either add or replace with python 3.6 as that's what we're using on `molo.core`